config.mk

```diff
@@ -22,6 +22,11 @@ ifeq ($(GAPPS_FORCE_MATCHING_DPI),false)
   endif
 endif
 
+GAPPS_TEST_LUNZIP := $(shell command -v lunzip)
```

Comment:
Fair enough. Since config.mk is only loaded once during build...maybe we can make it smart? Do something like

```
GAPPS_LZ_REQUIRED := $(shell find . -name '*.apk.lz' -print -quit)
ifneq ($(GAPPS_LZ_REQUIRED),)
// if not lunzip is installed, throw error
endif
```